demo/src/app/components/pagination/overview/pagination-overview.component.html

```diff
@@ -0,0 +1,44 @@
+<p>
+  Pagination is a component that only displays page numbers. It will not manipulate your data collection. You will have
+  to split your data collection into pages yourself.
+</p>
+
+
+<ngbd-overview-section [section]="sections['basic-usage']">
+  <p>
+    In order to properly operate a pagination behaviour, there are 3 important notions we have to be familiar with,
+    which are actually available as <code>@Input()</code> on the widget:
+  </p>
+  <ol>
+    <li><code>collectionSize</code> - Number of elements/items in the collection.
+      i.e. the total number of items the
+      pagination should handle.</li>
+    <li><code>pageSize</code> - Number of elements/items per page. i.e. the number
```

Comment:
> Number of elements/items per page. i.e. the number of items each page should contain

while ok for `collectionSize`, here for `pageSize` it sounds redundant

Reply:
Removed the unnecessary _ i.e. the number of items each page should contain_